docs/source/dev-guide/deep-dives/install.md

```diff
@@ -559,6 +559,8 @@ Their order is roughly how they happen in practice:
 7. `make_menu_action_groups`, composed of `MakeMenuAction` actions.
 9. `prefix_record_groups`, records installed packages in the environment via
    `CreatePrefixRecordAction` actions.
+10. `initial_action_groups`, includes any user-defined pre-transaction actions.
+11. `final_action_groups`, includes any user-defined post-transaction actions.
```

Comment:
```suggestion
10. `initial_action_groups`, includes any plugin-defined pre-transaction actions.
11. `final_action_groups`, includes any plugin-defined post-transaction actions.
```

Right? "user-defined" is technically true, but I think "plugin-defined" is a little more specific.